ui-admin/src/portal/siteContent/HtmlPageEditView.tsx

```diff
@@ -49,16 +98,41 @@ const HtmlPageView = ({ htmlPage, readOnly, siteInvalid, setSiteInvalid, updateP
     { renderAddSectionButton(0) }
```

Comment:
we could really use some extra padding at the bottom of the page to make the footer more accessible.  I'd suggest adding a `pb-4` or `pb-5` to the containing div on line 48